util/cron/common-oversubscribed.bash

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+#
+# Configure environment for testing in an oversubscribed manner (help multiple
+# chapel programs run nicer side by side)
+
+tasks=$($CHPL_HOME/util/chplenv/chpl_tasks.py)
+if [ "${tasks}" == "qthreads" ] ; then
+    export QT_AFFINITY=no
+    export CHPL_QTHREAD_ENABLE_OVERSUBSCRIPTION=1
+fi
```

Comment:
line 6 "tasks=..." and line 7 "if [ tasks ...]" are redundant when this is sourced from common-gasnet.bash or common-qthreads.bash (but not when sourced from test-xc-wb.bash).   I see it would be awkward to try and resolve that.